package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-lds",
-  "version": "7.0.0",
+  "version": "7.0.1",
```

Comment:
This is not a bugfix so should get `7.1.0` version imo

Reply:
just some minor stuff but kk

Reply:
https://semver.org/#spec-item-7 https://semver.org/#spec-item-6